client/src/components/DataFiles/DataFilesListing/DataFilesListing.test.js

```diff
@@ -243,6 +245,34 @@ describe('DataFilesListing', () => {
     fireEvent.change(dropdownSelector, { target: { value: 'Folders' } });
     expect(getAllByTestId('file-listing-item').length).toBe(4);
     fireEvent.change(dropdownSelector, { target: { value: 'All Types' } });
-    expect(getAllByTestId('file-listing-item').length).toBe(filesFixture.listing.FilesListing.length);
-  })
+    expect(getAllByTestId('file-listing-item').length).toBe(
+      filesFixture.listing.FilesListing.length
+    );
+  });
+
+  it('not rendering Shared Workspaces component', () => {
+    const history = createMemoryHistory();
+    history.push('/workbench/data/tapis/projects/');
+
+    systemsFixture.storage.configuration[5].hideSearchBar = false;
```

Comment:
I think a better test here would be to test that the Searchbar is not present when `hideSearchBar` is `true`